rust/processor/src/db/common/models/account_transaction_models/raw_account_transactions.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use crate::{
+    db::postgres::models::{
+        object_models::v2_object_utils::ObjectWithMetadata, resources::FromWriteResource,
+        user_transactions_models::user_transactions::UserTransaction,
+    },
+    utils::{counters::PROCESSOR_UNKNOWN_TYPE_COUNT, util::standardize_address},
+};
+use ahash::AHashSet;
+use aptos_protos::transaction::v1::{transaction::TxnData, write_set_change::Change, Transaction};
+use serde::{Deserialize, Serialize};
+
+pub type AccountTransactionPK = (String, i64);
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct RawAccountTransaction {
+    pub transaction_version: i64,
+    pub account_address: String,
+    pub block_timestamp: chrono::NaiveDateTime,
+}
+
+impl RawAccountTransaction {
+    /// This table will record every transaction that touch an account which could be
+    /// a user account, an object, or a resource account.
+    /// We will consider all transactions that modify a resource or event associated with a particular account.
+    /// We will do 1 level of redirection for now (e.g. if it's an object, we will record the owner as account address).
+    /// We will also consider transactions that the account signed or is part of a multi sig / multi agent.
+    /// TODO: recursively find the parent account of an object
+    /// TODO: include table items in the detection path
```

Comment:
Are these TODOs still relevant?

Reply:
Just for context, they're from this original PR: https://github.com/aptos-labs/aptos-core/pull/9076